### PR TITLE
Python3: Read file into a byte string

### PR DESCRIPTION
**Pull request by ziransun**

### CODE REVIEW

File: cookies/http-state/resources/cookie-setter.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 def dump_file(directory, filename):
   """Reads a file into a string."""
```
Comment (Hexcles) [inline cookies/http-state/resources/cookie-setter.py:28]:
```suggestion
  """Reads a file into a bytes string."""
```

File: cookies/http-state/resources/cookie-setter.py [hunk 2]
```diff
@@ -48,25 +48,25 @@ def body_with_expectation(self):
...
   string pair as used in the JavaScript object. Sorted by filename."""
   all_files = []
-  line_template = u'{{file: "{filename}", name: "{filename}"}},'
+  line_template = b'{{file: "{filename}", name: "{filename}"}},'
   for file in listdir(path.join(root, DEFAULT_RESOURCE_DIR, DEFAULT_TEST_DIR)):
     if file.endswith(u'-test'):
-      name = file.replace(u'-test', u'')
-      all_files.append(line_template.format(**{u'filename' : name}))
+      name = file.replace(u'-test', u'').encode()
+      all_files.append(line_template.replace(b"filename", name))
```
Comment (Hexcles) [inline cookies/http-state/resources/cookie-setter.py:61]: Shouldn't this be

```python
      all_files.append(line_template.replace(b"{filename}", name))
```
?

Also, I'd prefer to keep the template string as text and do the encoding here, e.g.

```python
      name = file.replace(u'-test', u'').
      all_files.append(line_template.format(filename=name).encode())
```

Comment (ziransun) [reply cookies/http-state/resources/cookie-setter.py]: Update. Thank you!

**Review by Hexcles — APPROVED** [review]
LGTM

**Pull request merged**